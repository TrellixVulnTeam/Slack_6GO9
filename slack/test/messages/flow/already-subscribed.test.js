const { AlreadySubscribed } = require('../../../lib/messages/flow');

describe('AlreadySubscribed message rendering', () => {
  test('works', async () => {
    const message = new AlreadySubscribed('atom/atom');
    expect(message.toJSON()).toMatchSnapshot();
  });
});
