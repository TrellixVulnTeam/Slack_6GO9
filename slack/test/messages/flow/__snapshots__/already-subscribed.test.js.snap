// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlreadySubscribed message rendering works 1`] = `
Object {
  "attachments": Array [
    Object {
      "color": "danger",
      "mrkdwn_in": Array [
        "text",
      ],
      "text": "You're already subscribed to \`atom/atom\`",
    },
  ],
  "response_type": "ephemeral",
}
`;
